.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
+# See https://help.github.com/articles/about-code-owners/
+
+/src/Common/src/System/Net/Http/Http2                    @dotnet/http2
+/src/Common/tests/Tests/System/Net/Http2                 @dotnet/http2
```

Comment:
@danmosemsft @maryamariyan are you fine having this CODEOWNERS file in addition to our area-owners?